orbit/cmd/orbit/orbit.go

```diff
@@ -341,7 +342,7 @@ func main() {
 		g.Add(systemChecker.Execute, systemChecker.Interrupt)
 		go osservice.SetupServiceManagement(constant.SystemServiceName, systemChecker.svcInterruptCh, appDoneCh)
 
-		if !c.Bool("disable-kickstart-softwareupdated") {
+		if !c.Bool("disable-kickstart-softwareupdated") && runtime.GOOS == "darwin" {
```

Comment:
Not clear why this is only logged for darwin.
Maybe add a comment ?

Reply:
`softwareupdated` is a macOS component only.

Reply:
So I guess this should have been named `disable-kickstart-softwareupdated-darwin` to begin with 🙂 
Maybe add a comment now like:
`//softwareupdated is a macOS component only.` ?